proto/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+pub mod market {
+    tonic::include_proto!("market"); // The string specified here must match the proto package name
+
+    use sha2::{Digest, Sha256};
+    use std::hash::{Hash, Hasher};
+    
+    // this is unnecessary I think
```

Comment:
Is this functionality used and the comment should be removed, or should the functionality be removed

Reply:
Should be removed, don't think we'll be using this with the std hash.